src/OpenTelemetry.AutoInstrumentation/Configuration/Settings.cs

```diff
@@ -167,6 +169,12 @@ private Settings(IConfigurationSource source)
     /// </summary>
     public bool Http2UnencryptedSupportEnabled { get; }
 
+    /// <summary>
+    /// Gets a value indicating whether the `AppDomain.UnhandledException` event should be enabled.
```

Comment:
nit: this is not markdown :) 
```suggestion
    /// Gets a value indicating whether the <see cref="AppDomain.CurrentDomain.UnhandledException"/> event should be enabled.
```